src/exporter.py

```diff
@@ -183,6 +189,10 @@ def start(self) -> None:
         """Start exporter service."""
         self._execute_service_action("start")
 
+    def running(self) -> bool:
```

Comment:
not-blocking: I think that using `is_running` name would be more self-explained. Also not as function, but as property?

Reply:
I agree that `is_running` is better name.
As for the property, I didn't feel good about hiding a subprocess call inside a property. I don't have strong opinion about it though. If you still think that property is better, I'll change it.